### ISSUE TITLE
Specified file glob seems to be ignored (on Windows)

### ISSUE THREAD

**Issue by sschuberth** [lycheeverse/lychee#961]
I'm running `~/Downloads/lychee-v0.11.1-windows-x86_64.exe --exclude-mail "**/*.md"` from Git Bash for Windows / MinTTY on https://github.com/oss-review-toolkit/ort and I'm getting

```
Error: Invalid path to URL conversion: reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl
```

Why does it complain about an *.ftl file when I instruct it to only check *.md files?

**Comment by lebensterben** [comment]
the errors comes from https://github.com/oss-review-toolkit/ort/blob/d0f9e69bf3c3a5c66a3dc583bbd689ff0584c835/docs/reporters/asciidoc-templates.md?plain=1#L13

**Comment by sschuberth** [comment]
Ah! But wouldn't it make sense to display (also) the path to the file that contains the broken link then?

**Comment by sschuberth** [comment]
BTW, on a rerun I'm now getting

```
thread 'tokio-runtime-worker' panicked at 'cannot send response to queue: SendError(Response(FsPath("README.md"), ResponseBody { uri: Uri { url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("www.tickgit.com")), port: None, path: "/browse", query: Some("repo=github.com/oss-review-toolkit/ort"), fragment: None } }, status: Timeout(None) }))', lychee-bin\src\commands\check.rs:160:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

which I didn't get before.

**Comment by lebensterben** [comment]
when reporting issues, please give your full command line input and the output.

**Comment by sschuberth** [comment]
> when reporting issues, please give your full command line input and the output.

The input is the same as [mentioned above](https://github.com/lycheeverse/lychee/issues/961#issue-1598188215). Here's all in one go:

```
$ ~/Downloads/lychee-v0.11.1-windows-x86_64.exe --exclude-mail "**/*.md"
thread 'tokio-runtime-worker' panicked at 'cannot send response to queue: SendError(Response(FsPath("README.md"), ResponseBody { uri: Uri { url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("github.com")), port: None, path: "/tools/godep", query: None, fragment: None } }, status: Timeout(None) }))', lychee-bin\src\commands\check.rs:160:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error: Invalid path to URL conversion: reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl
```

**Comment by mre** [comment]
My guess is that the error causes the channel to close.
We had issues with paths on Windows before. 
It's probably the same issue as in https://github.com/lycheeverse/lychee/issues/589.
So far it only got reported on Windows.

**Comment by mre** [comment]
This should be fixed with lychee 0.12.0.

```
PS C:\Users\mendler\Documents\ort-main> & '.\lychee-v0.12.0-windows-x86_64.exe'  --exclude-mail "**/*.md"
```

This works as expected on my Windows VM.
I also tried the previous version, and it crashed as you described.
Closing this, but feel free to reopen in case you still got any issues.
